Weather App/UI/HomeVC/HomeViewController.swift

```diff
@@ -29,8 +26,16 @@ class HomeViewController: UIViewController {
     
     private func addCallbacks(){
         
-        homeView.searchButtonPressed = searchPressed
-        homeView.settingsButtonPressed = settingsPressed
+        homeView.searchButtonPressed = {
```

Comment:
Refactor to call a method on the viewModel with the callback.

Reply:
I will add the methods.